resources/js/Pages/Praktikan.vue

```diff
@@ -2162,9 +2162,9 @@ export default {
     },
 
     formPassword: function($bool){
-      if($bool){
+      if($bool)
         this.viewPassForm = true;
-      }else{
+      if(!$bool){
```

Comment:
This one is still incorrect, please check back again of what i have suggested before @adhiiimf

Reply:
oooh i see wkwkwkw